check_runner/settings.py

```diff
@@ -0,0 +1,15 @@
+import configparser
```

Comment:
From codacy: missing a module docstring. I'll take a deeper look as to why the tox-run static code analysis didn't fail on that one.